src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -2151,6 +2152,120 @@ public class Testclass2 : ITest
             await TestWithPullMemberDialogAsync(testText, expected);
         }
 
+        [WorkItem(34268, "https://github.com/dotnet/roslyn/issues/34268")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public async Task TestPullPropertyToAbstractClassViaDialogWithMakeAbstractOption()
+        {
+            var testText = @"
+abstract class B
+{
+}
+
+class D : B
+{
+    int [||]X => 7;
+}";
+            var expected = @"
+abstract class B
+{
+    private abstract int X { get; }
+}
+
+class D : B
+{
+    int X => 7;
+}";
+            await TestWithPullMemberDialogAsync(testText, expected, selection: new[] { ("X", true) }, index: 1);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsPullMemberUp)]
+        public async Task PullEventUpToAbstractClassViaDialogWithMakeAbstractOption()
+        {
+            var testText = @"
+using System;
+namespace PushUpTest
+{
+    public class Base2
+    {
+    }
+
+    public class Testclass2 : Base2
+    {
+        private static event EventHandler Event1, Eve[||]nt3, Event4;
+    }
+}";
+            var expected = @"
+using System;
+namespace PushUpTest
+{
+    public abstract class Base2
+    {
+        private static abstract event EventHandler Event3;
```

Comment:
Hmm, abstract member can't be private or static. Are we intentionally breaking the code so the user will notice the error and fix them manually? Should we consider doing something automatically by default? like changing the accessibility to `protected  internal` and make it a non-static member.